src/main/java/dk/cngroup/lentils/controller/CypherController.java

```diff
@@ -1,13 +1,125 @@
 package dk.cngroup.lentils.controller;
 
+import dk.cngroup.lentils.entity.Cypher;
+import dk.cngroup.lentils.entity.Hint;
+import dk.cngroup.lentils.service.CypherService;
+import dk.cngroup.lentils.service.HintService;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.geo.Point;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
 
-@RestController
+import javax.validation.Valid;
+
+
+@Controller
 @RequestMapping("/cypher")
-public class CypherController
-{
-	private static final Logger LOGGER = LoggerFactory.getLogger(dk.cngroup.lentils.controller.CypherController.class);
+public class CypherController {
+    private static final Logger LOGGER = LoggerFactory.getLogger(CypherController.class);
+
+    private final String VIEW_CYPHER_LIST = "cypher/list";
```

Comment:
Tenhle endpoint tam vubec nepouzivas. list mas na endpointu "cypher/" a ne "cypher/list". Bud vybec jedno, nebo druhe.

Reply:
je to v 
 @GetMapping(value = "/")
    public String cyphers(Model model) {
        model.addAttribute("cyphers", cypherService.getAll());
        return VIEW_CYPHER_LIST;
    }

ale muzem si to vysvetlit. Rozumim tomu tak, ze "list" je jmeno view a to nemusi byt stejne s cestou v adrese "cypher/" nebo "cypher/list".  nebo musí / má ?